SR2024/2023-10-17-kickstart-reminder.md

```diff
@@ -0,0 +1,65 @@
+---
+to: Student Robotics 2024
+subject: Getting the most out of SR2024 Kickstart
+---
+
+Hello there!
+
+With [Kickstart][kickstart] this Saturday we wanted to let you know what to
+expect on the day and how to get the best start on your robot.
+
+As you already know, the event is fully remote. There are two main aspects to
+the day:
+
+- the presentation, which will be [livestreamed][livestream] on Youtube, and
+- the microgames, a set of tasks to familiarise you with the kit (details on
+  what you'll need below)
+
+We encourage all teams to watch the livestream on the day and work through the
+microgames together afterwards. If you are unable to do this for some reason the
+presentation and microgames will continue to be available afterwards on the
+[Kickstart event page][kickstart] and remain a great place for your team to
+start working on their robot.
+
+There will be two document links for the microgames -- one for the competitors
+which outlines the challenges and another for you which has the answers.
+
+What you'll need for the microgames:
+
+- **your kit**; all the kits have been dispatched -- please let us know if yours
+  hasn't arrived!
```

Comment:
We should confirm ourselves if we expect all kits to have arrived, particularly ones we dispatched this week.

Reply:
Issue: Not all kits have been dispatched quite yet (all bar 2, which gave us the details after the cutoff anyway).

Reply:
Ah, I'd misunderstood [your message earlier](https://studentrobotics.slack.com/archives/CMQ49PXPG/p1697475102293259) in Slack.

Reply:
Open to ideas for a better framing.

Reply:
@RealOrangeOne how much of a pain would it be to customise this line just for the two late teams?

Reply:
Thinking about this: since there's no personalisation here we can just do a BCC style send (rather than mail-merge), so doing two sends with slightly different content should be easy.